### PR TITLE
Added python scripts for cosine similarity, difflib.

### PR DESCRIPTION
**Pull request by Pilipets**
The purpose of this PR is to show small working scripts of:
1. Getting the syntactic difference between two text documents.
2. Getting cosine similarity among the set of the given documents using tf-idf approach and matplotlib for visualization.

Most likely difflib will be used from Golang because of performance issues.
We also need to cover the Russian language for both scripts.

### CODE REVIEW

File: ProjectDrafts/similarity_module.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    for i in range(len(y_labels)):
+        for j in range(len(x_labels)):
+            axes.text(j, i, "%.2f"%values[i, j],
+                      ha="center", va="center", color="w", fontsize=6)
+
+    fig.tight_layout()
+    plt.show()
+
+if __name__ == '__main__':
+    INPUT_FOLDER = sys.argv[-1]
```
Comment (gevorgyana) [inline ProjectDrafts/similarity_module.py:59]: We had better name it 'directory', but I am kidding :)

File: ProjectDrafts/similarity_module.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+def eprint(*args, **kwargs):
+    print('ERR %s: ' % datetime.now().strftime("%d/%m/%Y %H:%M:%S"), end='', file=sys.stderr)
+    print(*args, file=sys.stderr, **kwargs)
+    sys.exit()
+
+def cosine_similarity(file_objects, tokenizer_option):
+    remove_punct_dict = dict((ord(punct), None) for punct in string.punctuation)
+
+    if tokenizer_option == 0: # None
```
Comment (gevorgyana) [inline ProjectDrafts/similarity_module.py:16]: It is a bad idea to use numbers for indicating options, but this is a draft as your said.

**Review by gevorgyana — APPROVED** [review]
Okay as a draft, but we could avoid using numbers for option handling. I think there should be a library for parsing command line arugments easily, in Rust there is `clap`, I think there is something similar in Python. 

Anyway, thanks for reviving this project.

File: ProjectDrafts/similarity_module.py (hunk 2 above)
Comment (Pilipets) [reply ProjectDrafts/similarity_module.py:16]: Ok, I agree. Just wanted to do it quickly.

**Pull request merged**